app/models/user.rb

```diff
@@ -1,2 +1,17 @@
 class User < ApplicationRecord
+    
+    has_secure_password
+    
+    validates :name, presence: true
+    validates :email, presence: true, uniqueness: true
+    validates :password, presence: true
+
+    has_many :blogs
+    has_many :comments
+
+    def update_blogs_counter
+        self.blogs_counter = self.blogs.count
+        self.save
+    end
+    
```

Comment:
- [ ] When I test your API and create new blogs, the counter seems to not be working as expected and does not increment the blogs counter.To implement this, you can include the following method inside the posts model:

```
after_save :update_posts_counter_add

  def update_posts_counter_add
    user.increment!(:blogs_counter)
  end
```